WalletWasabi.Gui/Global.cs

```diff
@@ -264,17 +264,16 @@ public async Task InitializeNoWalletAsync(TerminateService terminateService)
 				}
 				else
 				{
+					var bestEffortEndpointConnector = new BestEffortEndpointConnector();
+					connectionParameters.EndpointConnector = bestEffortEndpointConnector;
```

Comment:
```suggestion
					connectionParameters.EndpointConnector = new BestEffortEndpointConnector();
```

Reply:
I need access to the instance. If i do what you say then I will need to cast the instances and that will look even worse.